src/app/app.module.ts

```diff
@@ -3,10 +3,16 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { SidenavComponent } from './layouts/sidenav/sidenav.component';
+import { NotFoundComponent } from './pages/not-found/not-found.component';
```

Comment:
All components must be inside the components folder for a neat folder structure. Moved the layouts and pages folder inside the components folder and also update all the imports.